crates/apps_lib/Cargo.toml

```diff
@@ -31,10 +32,13 @@ namada-eth-bridge = [
 
 [dependencies]
 namada_core = {path = "../core"}
+namada_io = { path = "../io" }
```

Comment:
maybe we could also re-export `namada_io` from sdk so we don't have to add this

Reply:
done